core/new-gui/src/app/dashboard/component/feature-container/saved-workflow-section/ngbd-modal-workflow-executions/ngbd-modal-workflow-executions.component.ts

```diff
@@ -93,6 +94,9 @@ export class NgbdModalWorkflowExecutionsComponent implements OnInit {
   ]);
   public showORhide: boolean[] = [false, false, false, false];
   public avatarColors: { [key: string]: string } = {};
+  public checked: boolean = false;
+  public setOfCheckedIndex = new Set<number>();
```

Comment:
please follow the same logic as https://github.com/Texera/texera/pull/1629/files